ci/compute-mx.jq

```diff
@@ -0,0 +1,36 @@
+def compute_arch($x):
+  ["amd64"] |
+  if
+    $x.LINUX_VER == "ubuntu20.04"
+  then
+    . + ["arm64"]
+  else
+    .
+  end |
+  $x + {ARCHES: .};
```

Comment:
```suggestion
def compute_arch($x):
  ["amd64"] |
  if
    ["ubuntu18.04", "centos7"] | any(index($x.LINUX_VER))
  then
    .
  else
    . + ["arm64"]
  end |
  $x + {ARCHES: .};
```

I think we should replace the `== "ubuntu20.04"`  with this suggested logic.

This way we don't have to explicitly add any new statements here when we add newer OSes, which will presumably have ARM support.